### ISSUE TITLE
Please add version to manifest.json

### ISSUE THREAD

**Issue by GSzabados** [garbled1/homeassistant_ecowitt#43]
Hello,

According to the latest breaking changes in HA 2021.3 you need to specify the version in the manifest.json for custom components.

https://www.home-assistant.io/blog/2021/03/03/release-20213/#breaking-changes

Could you please add it?

Thanks,

Gábor

**Comment by Tom7320** [comment]
Home Assistant will enforce this in the future:

`2021-04-10 02:02:27 WARNING (MainThread) [homeassistant.loader] No 'version' key in the manifest file for custom integration 'ecowitt'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'ecowitt'
`

**Comment by ronjtaylor** [comment]
Logger: homeassistant.setup
Source: setup.py:172
First occurred: 7:18:31 AM (1 occurrences)
Last logged: 7:18:31 AM

Setup failed for ecowitt: Integration not found.

I loaded 2021.6.0 this morning March 6 2021 and Ecowitt fails to load for this reason.
No Update in HACS yet.

**Comment by Tom7320** [comment]
Yes. I can confirm that the integration does not work anymore with 2021.6.0.

**Comment by bitlisz1** [comment]
Yepp, it's not working anymore. :(

**Comment by GSzabados** [comment]
To anybody who is unable to run this custom component with HA 2021.6.

As a temporary fix, you can add this to the manifest.json manually in your own setup:

`"version": "0.5",`

The manifest.json should be located in this place:

`/config/custom_components/ecowitt/manifest.json`

And should look like this after the manual modification:

{
  "domain": "ecowitt",
  "name": "Ecowitt Weather Station",
  "config_flow": true,
  "documentation": "https://www.home-assistant.io/integrations/ecowitt",
  "requirements": [
    "pyecowitt==0.11"
  ],
  "ssdp": [],
  "zeroconf": [],
  "homekit": {
  "dependencies": [],
  "version": "0.5",
  "codeowners": [
    "@garbled1"
  ]
}

The easiest to do this, is install File Editor from the Add-On store, and navigate to the file and change it manually.

This is just a temporary fix, until the PR will be merged. The same change will be available through HACS then.

**Comment by GSzabados** [comment]
Just a reference regarding the version number requirement:

https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions

**Comment by Tom7320** [comment]
> To anybody who is unable to run this custom component with HA 2021.6.
> As a temporary fix, you can add this to the manifest.json manually in your own setup:

Works! Thx a lot!

**Comment by bitlisz1** [comment]
> As a temporary fix, you can add this to the manifest.json manually in your own setup:
> 
> `"version": "0.5",`
> 
> The manifest.json should be located in this place:
> 
> `/config/custom_components/ecowitt/manifest.json`

Many thanks!

**Comment by edegraaff** [comment]
Changed  "version": "0.5", reboot, add integration, and up and running

**Comment by kcossabo** [comment]
thank you, made the modification, an can now ADD the integration after adding it to HACS

**Comment by GSzabados** [comment]
> thank you, made the modification, an can now ADD the integration after adding it to HACS
> 
> but, when adding I now get
> 
> Error
> Config flow could not be loaded

With HA 2021.8.X there is another breaking change regarding VOLT has to be changed to something else. Look at the other issues and PRs.

**Comment by alejayo** [comment]
> thank you, made the modification, an can now ADD the integration after adding it to HACS
> 
> but, when adding I now get
> 
> Error
> Config flow could not be loaded

It happens to me exactly the same.
